lib/src/main/java/io/ably/lib/realtime/Presence.java

```diff
@@ -296,27 +296,26 @@ public void unsubscribe() {
         eventListeners.clear();
     }
 
-
-    /***
-     * internal
-     *
-     */
-
     /**
-     * Implicitly attach channel on subscribe. Throw exception if channel is in failed state
-     * @param completionListener
-     * @throws AblyException
+     * Implicitly attach channel on subscribe. Throw exception if channel is in failed state.
+     * @param completionListener Registers listener, gets called when ATTACH operation is a success.
+     * @throws AblyException Throws exception when channel is in failed state.
      */
     private void implicitAttachOnSubscribe(CompletionListener completionListener) throws AblyException {
+        // RTP6e
         if (!channel.attachOnSubscribeEnabled()) {
             if (completionListener != null) {
-                completionListener.onSuccess();
+                String errorString = String.format(Locale.ROOT,
```

Comment:
I believe we can safely remove `Locale.ROOT` here

Reply:
Fixed -> https://github.com/ably/ably-java/pull/1043/commits/43f2eaa2a626be909a5f80fc9f159110c3b0c00c